js/util/combineCriteria.ts

```diff
@@ -0,0 +1,31 @@
+import * as Collections from 'typescript-collections';
+import { PropExtractorCriteria, PropExtractorConfig } from './extractProps';
+
+export const getCombinedCriteria = (
+  componentName: string,
+  configs: PropExtractorConfig[]
+): PropExtractorCriteria => {
+  let remainingConfigs: PropExtractorConfig[] = [...configs];
+  let inclusionSet = new Collections.Set<string>();
```

Comment:
The object is mutable, but the reference isn't, so this should be const.

Reply:
done - good catch